src/index.js

```diff
@@ -23,6 +23,7 @@ import React from 'react';
 import { createRoot } from 'react-dom/client';
 
 import 'patternfly/patternfly-4-cockpit.scss';
+import 'cockpit-dark-theme'; // once per page
```

Comment:
this is done few lines above; isn't it working already?

Reply:
That is actually true, how did I miss that bit?

Reply:
Hmm, is it the variant on my distro perhaps? i.e. it is outdated? I guess this can be closed then

Reply:
the work that @KKoukiou did is not in any released version so far; I guess we can do a new release of this in a month or two (this cockpit plugin does not get a lot of changes like others)

Reply:
That explains it, feel free to close it, thank you!

Reply:
No problem!

Which distro are you using, Fedora, RHEL, CentOS?